Corona-Warn-App/src/main/java/de/rki/coronawarnapp/diagnosiskeys/storage/CachedKeyInfo.kt

```diff
@@ -18,7 +20,7 @@ data class CachedKeyInfo(
     @ColumnInfo(name = "day") val day: LocalDate, // i.e. 2020-08-23
     @ColumnInfo(name = "hour") val hour: LocalTime?, // i.e. 23
     @ColumnInfo(name = "createdAt") val createdAt: Instant,
-    @ColumnInfo(name = "checksumMD5") val checksumMD5: String?,
+    @ColumnInfo(name = "checksumMD5") val etag: String?, // ETag
```

Comment:
Does it make sense to add some kind of getter for `etagWithQuotes` and `etagWithoutQuotes`?

Reply:
I've switched all code to treat etags as they are and not remove any quotes, special handling should no longer be required.

Reply:
Edge case would be invalidation during the first 14 days of 1.7.x beacuse it wouldn't match the etags we have already in db (etag as checksum without quotes).

Reply:
I don't think that should be a problem...